watch/watch.1

```diff
@@ -51,21 +64,29 @@ curses(3X)
 The
 .B watch(1)
 utility has a hazy history.
-Nobody seems to be actually sure of where it has appeared
-first.
+Nobody seems to be actually sure of where
+it has appeared first.
 .br
-According to procps 010114's watch C source code, where it first
-appeared in that package, it was written from scratch by Tony Rems
-in 1991, and later recieved many modifications and corrections by
+According to procps 010114's watch
+C source code, where it first appeared in
+that package, it was written from scratch
+by Tony Rems in 1991, and later recieved
+many modifications and corrections by
 Francois Pinard.
 .br
-However, according to Internet Initiative Japan Inc.'s iwatch
-manual page, the watch utility came first from BSDI Inc. BSD/OS 3.1
-and their code is slightly derived from it; he/she \(em who wrote the
-manual page and comments at the original source code \(em also
-speculated that its existence was earlier than BSD/OS 3.1, from some
-another free distribution, but there's no further information that
-supports this.
+However, according to
+Internet Initiative Japan Inc.'s iwatch
+manual page, the watch utility came first
+from BSDI Inc. BSD/OS 3.1 and their code
```

Comment:
BSDI Inc. isn't correct. It's simply BSDI (or later BSDi) Which stands for Berkeley Software Design, Inc.

Saying BSDI Inc is saying Berkeley Software Design Inc Inc.

I would put their later acronym without the extra `Inc.` suffix (BSDi with the lowercase `i`) to avoid confusion.

Reply:
I don't think it's consensus that RAS syndrome is incorrect

Reply:
```suggestion
from BSDi's BSD/OS 3.1 and their code
```

Like this? @mamccollum